src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/PageList.js

```diff
@@ -181,13 +187,26 @@ class PageList extends React.Component<Props> {
         }
     };
 
+    getIndicators = (item: Object) => {
+        const indicators = [];
+
+        if (!this.availablePageTypes.includes(item.template)) {
+            indicators.push(<Icon key="missing-template" name="su-exclamation-circle" />);
+        }
+
+        return indicators;
+    };
+
     render() {
+        const {getIndicators} = this;
+
         return (
             <div className={pageListStyles.pageList}>
                 <List
                     adapterOptions={{
                         column_list: {
                             display_root_level_toolbar: false,
+                            getIndicators,
```

Comment:
did you use came-case for this option on purpose? looks like we have not used that for the `display_root_level_toolbar` option

Reply:
Not on purpose, was just too eager to use the short hand notation :see_no_evil: